frontend/src/css/JoinWorkspace.styl

```diff
@@ -25,6 +25,12 @@
       height: 60px
       width: 100%
 
+      &:nth-child(2n):not(:hover)
+        background-color #f0f0f0
```

Comment:
Please, use the const at `frontend_lib/src/css/Variable.styl`

Reply:
done